src/main/java/com/skynest/uitesting/utils/BaseHelper.java

```diff
@@ -0,0 +1,8 @@
+package com.skynest.uitesting.utils;
+
+public class BaseHelper {
```

Comment:
make each method static and create private constructor so no instances can be created from this class